commit b688ffc29a685d1332e080798c914f26ae5c63c0
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Nov 10 16:52:15 2019 +0100

    Run interrupt handlers at spl7
    
    * i386/i386at/interrupt.S (interrupt): Call spl7 instead of the
    intpri-provided one.

diff --git a/i386/i386at/interrupt.S b/i386/i386at/interrupt.S
index cdb385c6..076e34bc 100644
--- a/i386/i386at/interrupt.S
+++ b/i386/i386at/interrupt.S
@@ -30,8 +30,7 @@ ENTRY(interrupt)
 	pushl	%eax			/* save irq number */
 	movl	%eax,%ecx		/* copy irq number */
 	shll	$2,%ecx			/* irq * 4 */
-	movl	EXT(intpri)(%ecx),%edx	/* get new ipl */
-	call	spl			/* set ipl */
+	call	spl7			/* set ipl */
 	movl	EXT(iunit)(%ecx),%edx	/* get device unit number */
 	pushl	%eax			/* push previous ipl */
 	pushl	%edx			/* push unit number */
